frontend/packages/db-structure/src/parser/__tests__/testcase.ts

```diff
@@ -1,35 +1,13 @@
 import {
   type Table,
   aColumn,
-  aDBStructure,
-  aTable,
+  type aDBStructure,
   anIndex,
 } from '../../schema/index.js'
 
-const userTable = (override?: Partial<Table>) =>
-  aDBStructure({
-    tables: {
-      users: aTable({
-        name: 'users',
-        columns: {
-          id: aColumn({
-            name: 'id',
-            type: 'bigserial',
-            notNull: true,
-            primary: true,
-            unique: true,
-          }),
-          ...override?.columns,
-        },
-        indices: {
-          ...override?.indices,
-        },
-        comment: override?.comment ?? null,
-      }),
-    },
-  })
-
-export const parserTestCases = {
+export const createParserTestCases = (
+  userTable: (override?: Partial<Table>) => ReturnType<typeof aDBStructure>,
```

Comment:
suggestion: It becomes more obvious if you do the following:

```suggestion
  userTable: (override?: Partial<Table>) => DBStructure,
```

Reply:
Oh, thank you so much.🙏
Fixed: https://github.com/liam-hq/liam/pull/583/commits/83736cfd8d6cca0eaf46d2582618d5bc1161be08